### ISSUE TITLE
Tooltips are not dismiss-able with escape key (with NVDA on)

### ISSUE THREAD

**Issue by schne324** [dequelabs/cauldron#586]
This appears to not be a valid issue. Test our tooltip component with NVDA to confirm that when a tooltip trigger is focused, and the tooltip is visible, pressing escape hides the tooltip.

If everything seems to work with NVDA on, please close this issue

---
<dl><dt>Description</dt><dd>Failure to make additional content dismissible without moving pointer hover or keyboard focus.
tool tip on below pages and respective elements  - 
1. issue panel > Edit, Remove state, Re-run automatic scan elements
2. Tooltip >  Text Tool-tips, Info Tool-tips and Bug tool-tips elements 
3. Tooltip Tab-stop > Demo elements
4. Pagination > Examples elements (at this time it is not overlapping but this can be reused in other products so it is recommended to provide a dismissible mechanism for all the tooltips.)  

Note: This issue should be taken care whenever additional content is being placed on other pages. controls. </dd><dt>Impact</dt><dd>Serious</dd><dt>Recommendation</dt><dd>RULE : 
When additional content is triggered on pointer hover or on keyboard focus, that additional content MUST be visually perceived and MUST be dismissible.

HOW TO FIX: 
Fix this issue by ensuring that content that appears on mouse hover or keyboard focus meets ALL of the following requirements:
1. Dismissible: If the content obscures or replaces other content, it can be dismissed via keyboard without moving mouse hover or keyboard focus - such as by pressing the Escape key.
2. Hoverable: If the content is triggered by mouse hover, the user can move the mouse pointer to the content without the content disappearing.
3. Persistent: The content stays visible until mouse hover or keyboard focus is moved or it is dismissed by the user (see #1 above), or the information is no longer valid.

REFERENCE: 
WCAG Understanding document: https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus

BACKGROUND: 
Content that appears on mouse hover or keyboard focus - such as tooltips, drop-down menus, and popups - can cause difficulties for people with disabilities. For people with low vision who use screen magnification, such content can be too large to fit in the magnified viewport or it may obscure other information on the screen. People with motor disabilities may have difficulty keeping the mouse pointer over the popup trigger. Sometimes, popup content is displayed only temporarily and may not give people with disabilities time to read the content. Content that appears on hover or focus should be dismissible, hoverable, and persistent. When these conditions are met, users have more control over interaction with the content.</dd><dt>Source code</dt><dd>

```html
<div id="tooltip73" class="Tooltip Tooltip--right Tooltip--hidden" role="tooltip" data-popper-reference-hidden="false" data-popper-escaped="false" data-popper-placement="right" style="position: absolute; inset: 0px auto auto 0px; transform: translate(314px, 273px);"><div class="TooltipArrow" style="position: absolute; transform: translate(0px, 13px); top: 0px;"></div>Edit</div>
```
</dd><dt>Checkpoint</dt><dd>Content on Hover or Focus (1.4.13.a)</dd><dt>URL</dt><dd>NA</dd></dl>

**Comment by scurker** [comment]
Tested on Windows 10 / NVDA 2021.3.5 and was unable to reproduce this issue. I was able to successfully dismiss tooltips when NVDA was running. The tooltip was announced along with the focused trigger.
